packages/lab/src/description-item/DescriptionRawItem.tsx

```diff
@@ -0,0 +1,41 @@
+import { Typography } from '@material-ui/core';
+import { Color, Column, Columns } from '@superdispatch/ui';
+import { forwardRef } from 'react';
+import styled from 'styled-components';
+
+export const DottedLine = styled.div`
+  border-bottom: 1px dashed ${Color.Silver400};
+  margin: 0px 8px;
+  height: 7px;
+`;
+
+interface DescriptionRawItemProps {
+  title: string;
+  value: string | number | JSX.Element;
+  boldValue?: boolean;
+}
+
+export const DescriptionRawItem = forwardRef<
```

Comment:
Lets move this component out of description item

Reply:
moved to `DescriptionDottedLineItem`